Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:lts-alpine
+ENV NODE_ENV=production
+WORKDIR /TuringTestGame
+COPY ["package.json", "package-lock.json*", "./"]
+RUN npm install --force

```

Comment:
I think we should use `npm install --legacy-peer-deps` instead of `--force`

Reply:
done